### DISCUSSION TITLE
Empty database after update

### DISCUSSION THREAD

**Discussion by majkinetor** [1Remote/1Remote#410] [General]
1. I took nightly archive
2. Turned off 1RM, copied the archive content over it, overwrite all
3. I started 1RM and empty database
    4. Database is shown in options as ./1remote.db
    5. I browse for database again 
    6. Now the full path is shown and it works.
    7. I go to global options again, and its exactly the same ./1remote.db path as before

@VShawn, looks like a bug ?

**Comment by VShawn** [comment]
It's a bug, but I can't reproduce it on debug version. I will delete the package and reopen #395

**Reply by VShawn** [reply]
And can not reproduce on my local release build.

**Reply by majkinetor** [reply]
It didnt happen on previous nightly if I remember correctly.

**Reply by majkinetor** [reply]
OK, moving this to bug then.
